### ISSUE TITLE
[Suggestion] Greatly increase honor gain.

### ISSUE THREAD

**Issue by silasftw** [Looking4Group/L4G_Core#3590]
I dont mind farming bgs, but right now it has been a pain, especially with always one BG up, if we make a premade horde stops queue and we dont get games and vice versa. And most bgs are very unfair. Without premade i played 4 hours yesterday without a single win, not even close ones. With premade easy 4cap wins. 

So during a whole weekend i think i got around 9k honor (did raid and some irls stuff) But even with full commitment a true pvper only get honor enough for approx 1 item. This would mean it would take like 3-4 months to just get fully geared for arena. On retails we all had this grind as well, it was a pain, but at least we had AV.s there and always a BG up 24/7 somewhere (thanks to battleground that was implemented).

Everywhere i go and talk with people they are getting really tired of this, not worth the time, some just thinking of stop playing. 

I would like the mod team to take a discussion about increasing honor gain with 2x 3x or 4x. People wants to go in to arena, and without resilience its just a zergfiesta. I know a lot of servers have increased honor gain. Since the items from honor dosnt give much advantage in pve it wont disturb the balance (later on some PVP gear is very legit in raiding).

I dont see much things that can backfire with multiplying the honor gain and maybe a community vote about it?. Especially now when we cant turn in tokens for honor (thats also an idea, to implement that quest). 

And my thread about being able to exchange tokens from one kind of BG to another still stands open (because of the lack of AV.s we cant get 4 items)

Cheers

**Comment by Krumbs56** [comment]
I agree with this. Either increase honor or let us turn in are marks for honor. Where is the daily BG quest anyway? I feel like the dev's are ignoring pvp : ( and this might scare people away.

**Comment by AnonXS** [comment]
But you know this Quest normally requires marks of alterac valley too, so we would need to change the quest requirements or would that be fine?

**Comment by silasftw** [comment]
i would combine it with my idea of being able to change your marks into AV ones, else we will NEVER be able to buy the gear.. which sucks... and as i pointed out - the gloves is one of the most important parts you can buy, we MAYBE get AV going during the weekend, but it wont be enough, and if someone cant play that weekend? Right now the PVP is draining the soul of enjoyment :( sadly., Whenever someone comes into discord and asks how the PvP is everyones reply is "dead" I mean sure if this is how you want it then thats it, but it will be a dead server regarding PvP, people have farmed BGs way to much in their past :P

Im prettyy sure you will have the same problem pve when people cleared all content, they want to play more! Challenge themself, everyone knows that the retail server kinda dies when its in the end of "before a patch" because there is nothing to do ingame :P

**Comment by silasftw** [comment]
I also want to add the fact with diminishing returns. In retail we had Battlegroups so you didnt meet the same people as much as you do know. Now after a couple of games you get 3-6 honor per kill, compared to maybe 30 which you should get. this is a HUGE factor, like this is where you get most honor from on weekends.

here is a great example: https://gyazo.com/1e63b9f33cfee9e4567c79e0108000f1

cant be done :( they even removed diminishing returns in BGs later on, and that even with different Battlegroups
